config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -2343,8 +2392,7 @@ public CsrfSpec accessDeniedHandler(ServerAccessDeniedHandler accessDeniedHandle
 
 		/**
 		 * Configures the {@link ServerCsrfTokenRepository} used to persist the CSRF
-		 * Token. Default is
-		 * {@link org.springframework.security.web.server.csrf.WebSessionServerCsrfTokenRepository}.
+		 * Token. Default is {@link WebSessionServerCsrfTokenRepository}.
```

Comment:
Please leave unrelated changes out of the PR. (It looks like a few others in this file sneaked in)